spyder/plugins/completion/kite/status.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+Status widget for Kite completions.
+"""
+
+# Local imports
+from spyder.config.base import _
+from spyder.utils import icon_manager as ima
+from spyder.widgets.status import StatusBarWidget
+
+
+class KiteStatus(StatusBarWidget):
+    """Status bar widget for Kite completions status."""
+
+    def __init__(self, parent, statusbar):
+        super(KiteStatus, self).__init__(parent, statusbar,
+                                         icon=ima.get_kite_icon())
+
+    def update(self, kite_enabled):
+        """Update kite completions status."""
+        self.set_value(_("Kite: enabled"))
+        self.setVisible(bool(kite_enabled))
```

Comment:
On VSCode they display if kite is not running (and kite is installed) the following:

<img width="578" alt="Screen Shot 2019-09-10 at 09 31 54" src="https://user-images.githubusercontent.com/3627835/64623047-e8cc0b00-d3ad-11e9-9cd6-6a3d830a3b77.png">

Maybe we should check if Kite is installed and running (display the current behavior) and if it is installed and NOT running something like the screenshot. And if kite is not installed then we hide the status widget (or not even create it?)

Also this needs to be a polling Status Widget, since Kite might be opened and the status will (and should change)

I guess Kite guys will like something similar @ccordoba12 ?

Reply:
> I guess Kite guys will like something similar

Ok, I agree. @dalthviz, please follow @goanpeca's suggestions.

Reply:
I agree with this, with the extra edge case:

if Kite is not detected as installed, but is enabled by the user (as per the option in #10169), we should probably continue to display this widget.